.meteor/packages

```diff
@@ -109,4 +109,5 @@ yasaricli:slugify
 yasinuslu:blaze-meta
 # sanjo:jasmine
 # velocity:html-reporter
+froatsnook:request
```

Comment:
I've used the npm module this is based off before, I wasn't aware of an http lib. I will check that out and then update this pull request. @rodrigok